vic/drivers/cesm/src/cesm_interface_c.c

```diff
@@ -76,6 +76,9 @@ int
 vic_cesm_init(vic_clock     *vclock,
               case_metadata *cmeta)
 {
+    global_param_struct global_param; 
+    option_struct       options;
+
```

Comment:
I don't think you want to / need to declare these here. They are in the header of this file.